cron_test.go

```diff
@@ -189,6 +189,35 @@ func TestRepeatLimit(t *testing.T) {
 	assert.Nil(t, cron.GetJob("test-repeat-limit"))
 }
 
+// Job with repeat limit in ISO8601.
+func TestRepeatWithISO8601(t *testing.T) {
+	calledCount := atomic.Int32{}
+
+	cron, err := New(
+		WithNamespace(randomNamespace()),
+		WithTriggerFunc(func(ctx context.Context, m map[string]string, p *anypb.Any) (TriggerResult, error) {
+			calledCount.Add(1)
+			return OK, nil
+		}))
+	if err != nil {
+		t.Fatal("unexpected error")
```

Comment:
Fatal with the error or use `require.NoError`.

Reply:
OK. it was copy-and-pasted from the other test case.

Reply:
Fixed all cases.